### ISSUE TITLE
Incorporating turns into the telemetry data

### ISSUE THREAD

**Issue by amritvignesh** [SCasanova/f1dataR#210]
Would be great if there can be a function to identify turns as it is hard to derive from coordinates itself!

**Comment by pbulsink** [comment]
@SCasanova We could just expose the `circuit_info` table from [FastF1](https://docs.fastf1.dev/circuit_info.html). 

@amritvignesh As much as possible we don't reprocess data from the Python package, it leaves this package much more complex. But FastF1 has that info available so you'd be able to put it together as needed (see from the FastF1 examples [circuit map with annotated corners](https://docs.fastf1.dev/examples_gallery/plot_annotate_corners.html) and also [speed trace with corner annotations](https://docs.fastf1.dev/examples_gallery/plot_annotate_speed_trace.html)).

**Comment by SCasanova** [comment]
Yes I think that could be useful. I will look into incorporating that this week
